src/searchCountry.js

```diff
@@ -0,0 +1,99 @@
+import axios from "axios";
+
+let countryName = "nederland"
+
+async function specificCountryData() {
+    let countryData;
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${countryName}`);
+        countryData = result.data.map((data) => {
+            return data;
+        })
```

Comment:
Dit is hetzelfde als:
```suggestion
        countryData = result.data;
```